appveyor.yml

```diff
@@ -12,20 +9,14 @@ branches:
 environment:
   global:
     CYG_ROOT: C:/cygwin
-    CYG_CACHE: C:/cygwin/var/cache/setup
-    CYG_MIRROR: http://mirrors.kernel.org/sourceware/cygwin/
-    OPAM_DIR: C:/opam
-
-cache:
-  - C:\opam
 
 install:
-  - set PATH=%OPAM_DIR%\bin;%PATH%
 #  - appveyor DownloadFile "http://cygwin.com/setup-%CYG_ARCH%.exe" -FileName "%CYG_ROOT%\setup-x86.exe"
-  - '"%CYG_ROOT%\setup-x86.exe" -qnNdO -R "%CYG_ROOT%" -s "%CYG_MIRROR%" -l "%CYG_CACHE%" -P make -P gcc-core -P ocaml -P ocaml-compiler-libs -P m4 -P tar -P patch -P flexdll -P pcre >NUL'
+  - ps: '(new-object net.webclient).DownloadString("https://aantron.github.io/binaries/cygwin/x86/ocaml/4.02/install.ps1") | PowerShell -Command -'
+  - ps: '(new-object net.webclient).DownloadString("https://aantron.github.io/binaries/cygwin/x86/camlp4/4.02/install.ps1") | PowerShell -Command -'
+  - ps: '(new-object net.webclient).DownloadString("https://aantron.github.io/binaries/cygwin/x86/opam/1.2/install.ps1") | PowerShell -Command -'
```

Comment:
Yes, it's possible to do it that way. However, the scripts are intended to be analogous to running `setup.exe`. They mean something like, "`install ocaml @ 4.02`", "`install opam @ 1.2`", etc. Cygwin doesn't seem to keep binaries around for old versions, and `setup.exe -q` doesn't even allow to select them. So, each of these scripts first downloads a little custom mirror, then installs the corresponding package from there. Since I don't think you would include an "official' Cygwin package containing a compiler, or `setup.exe` itself, into OASIS, I would argue that you shouldn't include these scripts or artifacts either.

I suppose if some change can't be made in the scripts upstream, for some reason, then they can be vendored into OASIS and modified. But, on the other hand, if I (or a contributor) discover that, e.g., the OCaml 4.02 build has some bug lurking, patch it, create a new 4.02 binary, and update the script to point to it, OASIS testing will immediately benefit from the patch if OASIS has not vendored the script. It's like a new revision of a package being uploaded to official Cygwin.

Reasons not to release the artifacts parallel to OASIS: they are for testing and assume AppVeyor, at least for the moment. And, each one is a binary package on the order of 10-50MB in size.

I should start documenting `binaries` :)